### PR TITLE
Add heroku deploy button readme

### PR DESCRIPTION
**Pull request by melveg**
The next changes includes the Heroku Deploy Button inside the `README.md` file with some instructions about this process and a `Procfile` to execute the web server.

I think this is ready for merging. But check it out first and let me know if you find something

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Heroku: ns-notifier",
+    "description": "A Flask App that send sms alerts about glucose level to your family or friends.",
+    "image": "heroku/python",
+    "repository": "https://gitlab.com/codeonrocks/client/nexmo-nsnotifier",
```
Comment (superdiana) [inline app.json:5]: This points to a private fork of this repository

File: README.md [hunk 2]
```diff
@@ -21,10 +21,20 @@ Although the initial state of this app is very basic, it's work in progress and
 
 ![Scout Dashboard](dashboard.png)
 
-## Contributing
+## Heroku Deploy
+
+Before you start deploying your App using the Heroku Deploy Button. You need to get the nexmo credentials from [Vonage dashboard](https://dashboard.nexmo.com/). Using the firebase console get the [Firebase database secrets](https://firebase.google.com/) and from google cloud get the [google client id](https://console.cloud.google.com/apis/credentials) needed for auth using the client. Two of this credentials are **Files**, the `firebase database secrets` and the `application private key`. You need to put them temporary in your repo to perform the deploy. After that you can remove them.
```
Comment (superdiana) [inline README.md:26]: There should be a way to not expose private keys or credentials.

Comment (superdiana) [reply README.md:26]: Perhaps the contents of those files to be pasted in those fields instead if those variables are left blank?

File: app.json (hunk 1 above)
Comment (melveg) [reply app.json]: @superdiana thanks for the observation, you are totally right. I changed this information in the last commit.

**Comment by superdiana** [comment]
Thank you very much @melveg for this awesome contribution!!

**Comment by melveg** [comment]
your welcome @superdiana , there is a remaining little change though. In the `README.md` in the deploy button section, template needs to point to the nexmo-community repo. Currently is pointing to my branch, i made this change temporary just for testing. But after merge `app.json` is available in master so the button should look like this now:

```
[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/nexmo-community/nexmo-scout)
```

**Pull request merged**